Scripts/RemediationScripts/ControlsEligibleForRemediationThroughUI.md

```diff
@@ -81,6 +81,8 @@
 
 40. [Azure_ServiceBus_AuthN_Disable_Local_Auth](ControlsEligibleForRemediationThroughUI.md#40-azure_servicebus_authn_disable_local_auth)
 
+41. [Azure_AppService_DP_Configure_EndToEnd_TLS](ControlsEligibleForRemediationThroughUI.md#40-Azure_AppService_DP_Configure_EndToEnd_TLS)
```

Comment:
This link on the UI on repo is not working?

Reply:
Done